assets/onboarding/fullscreen.scss

```diff
@@ -0,0 +1,60 @@
+
+.sensei-wp-admin-fullscreen {
+  background: #f6f7f7;
+  color: #50575e;
+
+  #wpwrap {
+    top: 0
+  }
+
+  #wpbody-content, #sensei-onboarding-page {
+    min-height: 100%;
+  }
+
+  #adminmenumain,
+  #wpcontent > *,
+  .error,
+  .notice,
+  .update-nag,
+  .updated,
+  #wpfooter {
+    display: none
+  }
+
+  #wpcontent {
+    margin-left: 0
+  }
+
+  #wpcontent > #wpbody {
+    display: block
+  }
+
+  * {
```

Comment:
This is mostly for our own CSS and things we put on this page, so the only thing it can affect that is outside our control is the Wordpress and Woocommerce components we use. Some of those already have `border-box` set up in their component styles, and both Gutenberg and Woocommerce-admin's CSS resets include it, so this is their expected environment.

Reply:
Hum! I got! In this case, what do you think to move it to the `assets/onboarding/onboarding.scss` under the `.sensei-onboarding` scope? Does it make sense to you? Or could it not cover a case?

Reply:
I was thinking about this too, but it led to a bit of refactoring. I added a new `sensei-onboarding__page` class to the body, and extended the `useFullscreen` hook to support this. (Might need a better name)

We need this because `Modal` components are rendered outside the app and attached directly to the body, so anything scoped for our original `sensei-onboarding` container would not affect them.

Reply:
Hum! I got your point! So I think we can go ahead with this and if we see an error in the future we can investigate some workaround based on the error.

And I liked your approach to make to hook dynamic. =)